medusa/service/grpc/medusa.proto

```diff
@@ -71,11 +73,22 @@ message BackupSummary {
   int32               finishedNodes = 5;
   repeated BackupNode nodes = 6;
   StatusType    status = 7;
+  string    backupType = 8;
 }
 
 message BackupNode {
   string         host = 1;
   repeated int64 tokens = 2;
   string         datacenter = 3;
   string         rack = 4;
+}
+
+message PurgeBackupsRequest {
+}
+
+message PurgeBackupsResponse {
+  int32     nbBackupsPurged = 1;
+  int32     nbObjectsPurged = 2;
+  int64     totalPurgedSize = 3;
+  int32     totalObjectsWithinGcGrace = 4;
```

Comment:
Would a `GMT date-time` inclusion be useful for when the purge succeeded for the given totals?